.github/workflows/integration_tests.yml

```diff
@@ -70,16 +70,24 @@ jobs:
 
       - name: Install Keptn
         id: install_keptn
-        uses: keptn-sandbox/action-install-keptn@v1.0.0
+        uses: keptn-sandbox/action-install-keptn@v2.0.0
         timeout-minutes: 10
         with:
           KEPTN_VERSION: ${{ matrix.keptn-version }}
-          KEPTN_INSTALL_PARAMETERS: -v --endpoint-service-type=LoadBalancer
+          HELM_VALUES: |
+            control-plane:
+              apiGatewayNginx:
+                type: LoadBalancer
+              features:
+                automaticProvisioningURL: http://keptn-gitea-provisioner-service.default # Keptn 0.16 compatability 
+                automaticProvisioning:
+                  serviceURL: http://keptn-gitea-provisioner-service.default
```

Comment:
Please note: You have to move the keptn isntall step AFTER you installed gitea provisioner service